example_notebooks/example_files/CP_files/mitocheck_idr_cp.cppipe

```diff
@@ -32,7 +32,7 @@ Metadata:[module_num:2|svn_version:'Unknown'|variable_revision_number:6|show_win
     Regular expression to extract from folder name:(?P<Date>[0-9]{4}_[0-9]{2}_[0-9]{2})$
     Extract metadata from:All images
     Select the filtering criteria:and (file does contain "")
-    Metadata file location:Default Input Folder|Desktop/Github/IDR_stream/example_files
+    Metadata file location:Default Input Folder|Desktop/Github/IDR_stream/example_notebooks/example_files
```

Comment:
I just want to make sure...

If someone uses `idr_stream_cp` outside of the `Desktop/Github/IDR_stream/example_notebooks/example_files` this will not break IDR_stream CP will it? Ex: the code for `mitocheck_data` repo is running in a completely different location.

Reply:
I would recommend not just looking at the `.cppipe` file as it is a CellProfiler-specific file format that displays all of the parameters in each module in the pipeline regardless of whether it is used or not. This is especially important if you are not familiar with the pipeline as the CellProfiler GUI makes it more readable in this case.

To give an explanation of what is going on in the `.cppipe` file:

For that part of the pipeline you highlighted, you can see there is a path next to the `Metadata file location` parameter. But, you can see that before this path, the parameter is set to Default Input Folder, which will be whatever you have set for your CellProfiler (for example, on mine it defaults as `/home/jenna`). The path next to the parameter is to the metadata file that I used to confirm that the pipeline is working with example images when I had this parameter set to Default Input Subfolder. CellProfiler saves any path you give for a parameter even if you don't use it. If you open up this file in the GUI, you can see I have the file location set to the Default Input Folder with no path so that it won't try to look into the images directory with that path. 

To answer your specific question, as long as an individual when using idr_stream CP gives the correct metadata path (regardless of where the file is) so that it can be copied into the temporary folder, then it will not break since CellProfiler CLI expects the metadata file to be in the same directory as the images.

Let me know if that doesn't make sense or if you have any more questions about CellProfiler `.cppipe` files :smile:

Reply:
Sounds good, thanks for the clarification!